tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1584,6 +1584,12 @@ lldp/test_lldp.py::test_lldp_neighbor:
 #######################################
 #####           macsec            #####
 #######################################
+macsec:
+  skip:
+    reason: 'Skip macsec test for T2, if macsec is not enabled'
+    conditions:
+      - "macsec_en==False and 't2' in topo_type"
```

Comment:
If macsec_en==False, then it should be skipped, we shouldn't assert and restrict for T2 here